storage_image/views/storage_image_relation_abstract.xml

```diff
@@ -46,15 +46,44 @@
             <form>
                 <div class="row o_storage_image_relation">
                     <div class="col-md-6 col-xs-5">
-                        <group name="image">
-                            <field name="image_id" />
+                        <group
+                            name="action"
+                            attrs="{'invisible': [('id', '!=', False)]}"
+                        >
+                            <field name="id" invisible="1" />
+                            <field name="add_type" string="What do you want to do?" />
```

Comment:
```suggestion
                            <field name="add_type" widget="radio" options="{'horizontal': True'}" nolabel="1" />
```

This might look a bit better IMO

Reply:
:+1: